src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+import { PersonInfo } from '../PersonInfo';
+
+interface Props {
+  people: Person[];
+  onSelect: (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    person: Person,
+  ) => void;
+}
+
+export const Dropdown: React.FC<Props> = ({ people, onSelect }) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      {people.length ? (
+        <div className="dropdown-content">
+          {people.map(person => (
+            <PersonInfo person={person} key={person.slug} onSelect={onSelect} />
```

Comment:
Ensure that the `PersonInfo` component correctly handles the `onSelect` prop. This is important for the dropdown functionality to work as expected when a person is selected.